m4opt/missions/_lsst/data/README.rst

```diff
@@ -0,0 +1,19 @@
+Reproduced Data from lsstCamSim.yaml
+====================================
+
+The contents of this directory are reproduced from the following source:
+`lsstCamSim.yaml <https://github.com/lsst/obs_lsst/blob/main/policy/lsstCamSim.yaml>`_.
+
+1. Extracting data from `lsstCamSim.yaml` is computationally expensive.
+   To optimize processing, we extract only relevant information, focusing on science detectors
+   while excluding wavefront and guide sensors. The remaining 189 detectors are stored in an
+   `astropy.table.Table` as a CSV file (`lsst_science_detectors.csv`), which contains the LSST camera geometry.
+
+2. This CSV file is read by `../camera.py` to construct the LSST Field of View (FOV).
```

Comment:
I don't see a copy of this file in the obs_lsst repository. Is this a data product that you have generated? If so, then please include your source code.

I am optimistic that we can make that code run fast enough that we just use that, and we don't have to include the CSV file at all.